pom.xml

```diff
@@ -212,6 +213,11 @@
         <artifactId>aws-java-sdk-s3</artifactId>
         <version>${aws.amazonaws.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.github.ben-manes.caffeine</groupId>
```

Comment:
double checking, what's the license type of new dependency library?

Reply:
Apache 2.0
Just for reference, the reason I added this is the current caching library used in Alluxio mentions in the java doc that you should use this updated one https://github.com/google/guava/blob/master/guava/src/com/google/common/cache/CacheBuilder.java#L46

Reply:
After doing some Microbenchmark performance testing on this cache I decided to move back to the google guava cache for now, but more investigation is needed in the future https://tachyonnexus.atlassian.net/browse/AE-2640